plugin/src/main/java/io/jenkins/plugins/remotingkafka/KafkaComputerLauncher.java

```diff
@@ -38,6 +40,8 @@
 public class KafkaComputerLauncher extends ComputerLauncher {
     private static final Logger LOGGER = Logger.getLogger(KafkaComputerLauncher.class.getName());
     private static final long DEFAULT_TIMEOUT = 60000;
+    private static final String K8S_AGENT_CONTAINER_NAME = "agent";
+    private static final String K8S_AGENT_CONTAINER_IMAGE = "jenkins/remoting-kafka-agent:latest";
```

Comment:
Would be nice to make this parameterized; suggest doing that in a follow up change